scripts-dev/lint.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
this should probably have been a separate PR...

Reply:
you're right... (is this your script?) I love it too much, I couldn't resist (I also notice it's grown the difference option since last time I looked at it :))

will move over